var/spack/repos/builtin/packages/casacore/package.py

```diff
@@ -73,3 +76,7 @@ def cmake_args(self):
 
         args.append('-DBUILD_TESTING=OFF')
         return args
+
+    def patch(self):
+        os.remove('cmake/FindHDF5.cmake')
+        return
```

Comment:
```suggestion
    def patch(self):
        # Rely on CMake ability to find hdf5, available since CMake 3.7.X
        os.remove('cmake/FindHDF5.cmake')
```